### ISSUE TITLE
'DataFrame' object has no attribute 'append'  

### ISSUE THREAD

**Issue by adityamalikzeiss** [AutoViML/Auto_TS#113]
I am getting the following exceptions while trying Auto_TS:

Exception occurred while building Prophet model...
'DataFrame' object has no attribute 'append'
    FB Prophet may not be installed or Model is not running...

Exception occurred while building Auto SARIMAX model...
'DataFrame' object has no attribute 'append'
    Auto SARIMAX model error: predictions not available.

I have followed the example notebook given for univariate model.

**Comment by adityamalikzeiss** [comment]
Getting the exact same errors when executing the autots_univariate_example.ipynb:

Best model is a Seasonal SARIMAX(2,1,1)*(0,0,0,12), aic = 2458.432
Exception occurred while building Auto SARIMAX model...
'DataFrame' object has no attribute 'append'
    Auto SARIMAX model error: predictions not available.

Fold Number: 2 --> Train Shape: 441 Test Shape: 5
Exception occurred while building Prophet model...
'DataFrame' object has no attribute 'append'
    FB Prophet may not be installed or Model is not running...

**Comment by BryR0** [comment]
replace line models\build_*.py:
 **extra_concatenated = extra_concatenated.append(concatenated)**
   
 to 
`**try:

   extra_concatenated = extra_concatenated.append(concatenated)
   
except Exception as e:

  extra_concatenated = pd.concat([extra_concatenated,concatenated])** `
              
 in line 968 auto_ts\__init__.py:
 ##### Now make predictions using model given ######
predictions=None <---- local variable

**Comment by AutoViML** [comment]
yes I am aware that the new pandas has deprecated the append statement. I will try to upgrade auto-ts soon with pandas 2.0. 

Currently there are problems between dask and pandas that it may be difficult to upgrade one vs. other.

Ram
